content/messaging/index.mdx

```diff
@@ -88,6 +88,30 @@ To learn about differences between these two API types, see the comparison below
 
 If you're not sure, which implementation to choose, we suggest reading about particular APIs. You'll find the **When to use RTM/Web API** section in each document - refer to the table above for links.
 
+## API Versioning
+
+LiveChat Messaging API doesn't follow <a href="hhttps://semver.org/" target="_blank">semantic versioning</a>. We suggest you learn our system and terminology:
```

Comment:
I'm not sure if we should write about not following semver :/ It can sound negative, I don't know.

Reply:
Is it a bad thing we're not following it? If it's not, I would leave this statement. At least we're transparent and we don't mislead developers in this area.

Reply:
ok

Reply:
I mean few apis follow semver, it was introduced mainly for libraries or applications, not online apis

Reply:
@Andarist, what's your opinion on this?

Reply:
@bkubiak is right that platform APIs usually do not adhere to semver. Ultimately it doesn't matter much, the important part is to describe the versioning policy accurately. We don't have to mention semver explicitly, but definitely we need to mention that breaking changes might get introduced between version (which we do). Maybe a good thing that would help to emphasize this would be a short paragraph about API upgrades? Like - when you should do it, what you should check when doing so etc.

A stable~ release schedule would be also helpful for developers, something like in case of Shopify and others:
<img src="https://cdn.shopify.com/s/files/1/0533/2089/files/api-versioning-at-shopify-locked-calendar-year.png?format=webp&v=1554732991"/>

Reply:
thank you! I suggest we kick off the semver mention, publish the content as it is, and develop on upgrades in the second iteration. Some visuals would be great, I can arrange for that to be taken care of.